### ISSUE TITLE
[Feature]: Piped - Add song to piped playlist, auto-sync playlist and liked songs

### ISSUE THREAD

**Issue by SerAn51** [fast4x/RiMusic#2651]
### Description

The only piped thing currently working is the playlist synching but not the songs inside (more details below).
When adding a song to a piped playlist, that song doesn't been added to Piped but only locally the piped playlist (logging into Piped main website you'll see no song).

For the liked songs, it will be awesome if they were synched with Piped automatically if the user had done the log in with Piped credentials.

NB: if a song is added directly from piped website or another piped service, the playlist in RiMusic update itself correctly anche the new song is correctly showned in the piped playlist. So the problem with the songs is exclusively relating to a missing "sync with piped when adding a song" feature.

### Suggested Solution

(1) When adding a song to a piped playlist it will been added in the piped account for that playlist.
(3) Launching the app, the liked songs sync with Piped.
Basically, it will be awesome to sync everything automatically.

Thx for all the work, with this implementations piped will be 100% covered I suppose.
